### DISCUSSION TITLE
How's cxx's options field on testing pugixml ?

### DISCUSSION THREAD

**Discussion by parsanoori** [Homebrew/discussions#2192] [Writing Formulae/Casks] [answered]
Consider [this line](https://github.com/Homebrew/homebrew-core/blob/797bbae2e9ef8858cf4459fb8a32488cbbd506f1/Formula/pugixml.rb#L49) in pugixml formula. For ease of access I copy it here too:
```
    system ENV.cxx, "test.cpp", "-o", "test", "-I#{include}",
                    "-L#{lib}", "-lpugixml"
```
How are `include` and `lib` filled ?
Since I can't find them in any other place in the formula.

**Comment by SMillerDev** [comment]
They are the standard locations for these for these things, defined in brew itself.

**Reply by parsanoori** [reply]
How can I figure out these values ?

**Comment by carlocab** [comment, marked as answer]
`include` is just `$HOMEBREW_PREFIX/opt/pugixml/include` and `lib` is `$HOMEBREW_PREFIX/opt/pugixml/lib`.

**Reply by gromgit** [reply]
@parsanoori, you should probably also read https://docs.brew.sh/Formula-Cookbook#variables-for-directory-locations.
